src/PSR7Session/Http/SessionMiddleware.php

```diff
@@ -94,7 +101,8 @@ public function __construct(
         SetCookie $defaultCookie,
         Parser $tokenParser,
         int $expirationTime,
-        int $refreshTime = self::DEFAULT_REFRESH_TIME
+        int $refreshTime = self::DEFAULT_REFRESH_TIME,
+        ClockInterface $clock = null
```

Comment:
This means moving the clock up in the constructor parameters, and then we'd add a `CHANGELOG.md` entry documenting the BC break